src/app/site/cart/components/cart-items/cart-items.component.html

```diff
@@ -27,25 +27,32 @@ <h4><a [routerLink]="['/', 'product', product?.product_id, product?.color._id]">
         </div>
         <div class="item-data">
           <div>
-            <p>اندازه:</p>
-            <span>{{displaySize}}</span>
+            <p style="color: grey;">سایز:</p>
+            <span style="position: relative;left: 9px;">{{displaySize}}</span>
           </div>
+          <section><p style="color: grey;">تعداد:</p><span style="position: relative;left: 20px;">{{displayQuantity}} {{' عدد'}}</span></section>

```

Comment:
please use exact color code: `#8f8f8f` (and if possible, use the color property at top level instead of inline styling of many elements)